register.py

```diff
@@ -1,14 +1,17 @@
 import pypandoc
-import os
 
 output = pypandoc.convert('README.md', 'rst')
-f = open('README.txt','w+')
+f = open('README.txt', 'w+')
```

Comment:
Shouldn't this use a `with` statement (also for consistency since `with` is also used 12 lines below)?

```python
with open('README.txt', 'w+') as f:
    f.write(output)

```